src/components/Bag.jsx

```diff
@@ -0,0 +1,131 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Popover, Text, Button } from "@sparrowengg/twigs-react";
+import { removeFromCart, setTotalAmount } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useMemo } from "react";
+const Bag = () => {
+  const cart = useSelector((state) => state.cart);
+  const totalAmount = useSelector((state) => state.totalAmount);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    let tot = 0;
+    cart.forEach((ele) => {
+      tot = tot + Number(ele.cartCount) * Number(ele.price);
+    });
+    tot = Number(tot.toFixed(3));
+    dispatch(setTotalAmount(tot));
+  }, [cart]);
+
+  // const bagTotal = useMemo(() => {
+  //   let tot = 0;
+  //   cart.forEach((ele) => {
+  //     tot = tot + Number(ele.cartCount) * Number(ele.price);
+  //   });
+  //   return tot.toFixed(2);
+  // }, [cart]);
+  return (
+    <>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-between"
+        gap="10px"
+        css={{ width: "300px", overflow: "scroll", height: "100%" }}
+      >
+        <Text
+          css={{
+            marginTop: "30px",
+            marginBottom: "30px",
```

Comment:
@arun-121 this can be marginBlock: "$15"

Reply:
changed as mentioned